library/dokku_network_property.py

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# from ansible.module_utils.basic import *
+from ansible.module_utils.basic import AnsibleModule
+import subprocess
+
+DOCUMENTATION = """
+---
+module: dokku_network_property
+short_description: Set or clear a network property for a given dokku application
```

Comment:
Could you include an example of clearing a network property?

Reply:
There actually is already an example - please see _"De-associate the container with the network."_

However, I understand that switching between the expressions _"assoication a network"_ and _"setting a network proprety"_ is a bit confusing. However, I tried to stick to the official dokku documentation when it comes to that.

What do you think about changing **"De-associate the container with the network."** to _"Clearing a network property (de-associate the container with the network)."_?

Reply:
There is  already an example (see _De-associate the container with the network._) but I will rename it to make it more clear